src/dogeApi.js

```diff
@@ -0,0 +1,80 @@
+const axios = require('axios')
+const settings = require('./settings')
+
+const DOGE_NODE_URL = `http://${settings.RPC_HOST}:${settings.RPC_PORT}`
+const BASIC_AUTH_DOGE_TOKEN = Buffer.from(`${settings.RPC_USER}:${settings.RPC_PASSWORD}`).toString('base64')
+
+function getAccountAddress (account) {
+  return axios.post(DOGE_NODE_URL, {
+    jsonrpc: '2.0',
+    id: + new Date(),
```

Comment:
A quoi ça sert ?

Reply:
It is just converting it into a timestamp. And we use it for the `id` in the JSON-RPC request.

see this example. Pretty neat trick.
```javascript
> new Date()
2021-04-17T13:54:53.274Z
> + new Date()
1618667699856
```